learn/appchains/index.md

```diff
@@ -1,9 +1,20 @@
 ---
 title: Appchain
 description: Discover the features in EVM and Substrate appchains deployed through Tanssi, leveraging user-friendly and easy-to-deploy templates for enhanced capabilities.
-template: subsection-index-page.html
-hide: 
- - toc
- - feedback
+icon: simple-flathub
+template: index-page.html
 ---
 
+# Appchains In Tanssi
+
+Developing a blockchain that balances scalability, efficiency, and customization is a complex task. Tanssi appchains address these challenges through their integration into Tanssi's modular infrastructure. This section explains the structure, functionality, and unique characteristics of appchains in the Tanssi ecosystem.
+
+Key features of appchains in Tanssi:
+
+- **Runtime customization** - adapt transaction types, execution logic, and state transitions to fit your appchain’s specific requirements
+- **Transaction efficiency** - optimize transaction fees for scalability and economic sustainability
+- **Scalability at its core** - build appchains that are prepared to grow alongside your ecosystem
+
+## Explore What This Section Covers
+
+:::INSERT_GENERATED_CARDS:::
```

Comment:
When on this page, the icons don't look like they're from the same family. Overview and Core Runtime features are similar, but the Core Runtime features icon looks like it has a thicker line stroke. The transaction fees looks totally different, because the other two don't have filled in elements, and this one does

<img width="1780" alt="Screenshot 2024-12-16 at 12 24 42 AM" src="https://github.com/user-attachments/assets/e0e50fa7-3b84-45ab-a387-01e8b7f740d0" />

Reply:
change core features to octicons family to have the same line thickness and in trasacction fee change to outline material so that it has no filling and has a coherence with the others.

Reply:
Maybe we can change the transaction fee icon for this one, `law-24.` What do you think?